pytanque/protocol.atd

```diff
@@ -23,23 +23,27 @@ type request = {
     params: abstract
 }
 
-type init_params = {
-    ~debug <python default="False">: bool;
-    root: string;
-}
-
 type start_params = {
-    env: int;
     uri: string;
     thm: string;
 }
 
+type run_ops = {
+    ~memo <python default="True">: bool;
+    ~hash <python default="True">: bool;
+}
+
 type run_params = {
+    ?opts: run_ops option;
     st: int;
     tac: string;
 }
 
-type run_response = [ Current_state of int | Proof_finished of int ]
+type state = {
```

Comment:
We still call this run_result

Reply:
Yes, but I find `State` more intuitive since it is also the input type for most commands.